.gitignore

```diff
@@ -280,3 +280,4 @@ __pycache__/
 # Cake - Uncomment if you are using it
 # tools/**
 # !tools/packages.config
+korebuild-lock.txt
```

Comment:
I think the file name is a matter of preference. I prefer to have the file extension .txt. In the event we decide to add more to the lockfile or change its format, it could become korebuild-lock.{json,xml,csv,yml}.